.github/workflows/continuous_integration.yml

```diff
@@ -199,8 +205,10 @@ jobs:
           filters: |
             other_than_docs:
               - '!(docs/**)**'
+            other_than_timeseries:
+              - '!(timeseries/**)**'
       - name: Test Multimodal Others on AWS Batch
-        if: steps.changes.outputs.other_than_docs == 'true'
+        if: steps.changes.outputs.other_than_docs == 'true' && steps.changes.outputs.exclude_timeseries == 'true'
```

Comment:
Needs to be `steps.changes.outputs.other_than_timeseries`, ditto for other instances

Reply:
made the changes, sorry for the oversight